internal/image/validate_test.go

```
@@ -708,3 +708,122 @@ func TestValidateImageSkipImageSigCheck(t *testing.T) {
 		})
 	}
 }
+
+func TestValidateImageSkipAttSigCheck(t *testing.T) {
+	tests := []struct {
+		name                   string
```

golangci-lint (2.12.2)

[error] File is not properly formatted

(gci)